attributes/default.rb

```diff
@@ -1 +1,25 @@
+# 'initial_url_required' is there to preserve backwards compatable
+# behavior with the original Rackspace version
+# Override it to false to not require 'initial_url' in your
+# environment, role or wrapper cookbook
+default['scaleft']['initial_url_required'] = true
```

Comment:
I'm open, I put that in just to make it backwards compatible with what you folks were doing. I always set 'initial_url_required' to false.

Its not stated in the docs, but if you are using the SaaS you don't need to specify the initial_url, it seems to default to the right url. Seems to be only needed if you are **not** using the SaaS. 

We don't set the `initial_url` on ours, we don't explicitly set auto-enroll (defaults to it) We just set the CannonicalName and the Bastion (and maybe the AltNames). We also associated an AWS Account with each project but that is thru the Web UI.

My git-fu isn't up to knowing how to squash the comments at this point. 

Let me know if you want me to update the Pull Request by taking out all the logic around `initial_url_required` 

It seems to me to be difficult and maybe unesseccary to have logic as @dude051 proposed. This is because:

`auto_enroll` is `true` by default if not specified in the `sftd.yml` file, and if the `initial_url` attribute is `nil` or `false` in the cookbook. Also as I mentioned if you are using the SaaS you do not need specific the `initial_url`

I had assumed that the original cookbook had the requirement for `initial_url` being set because folks at Rackspace was using a Private version of scaleft and not the SaaS. In which case requiring it to be set made sense, That is why I made the original behavior the default and had an explicit mechanism to override that. 

Don't know if there is any way to programmatically sense if you had a private ScaleFt service and then require the `initial_url` to be set.